src/Json.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.9.0;
+
+import "console2.sol";
+import "Vm.sol";
+
+// Helpers for parsing keys into types.
+contract JsonParser {
```

Comment:
What do you think about this?

```solidity
using JsonParser for string;

function test() public {
    string memory someJson = vm.readFile("...");
    address someAddr = someJson.readAddress("...");
    // ...
}
```

Also, we should choose that is in line with https://github.com/foundry-rs/forge-std/pull/157. I suggest `stdJson` if it is going to be a library, and `StdJson` otherwise. We can improve the function names as necessary.

Reply:
Yea I like this, making it a lib definitely feels like the right approach and should be doable:
- The current `path` storage var is just used to alias `readFile` to `readJson`. I'd argue this isn't necessary, but could add a helper method to the lib if we like the explicit naming.
- By making `string memory json` the first arg for all methods, we don't need the `json` storage var and can use the `using for` syntax @ZeroEkkusu has above.
- I think we can remove the `console2` import/usage, as that's easy for users to implement themselves and doesn't add much value IMO